frontend/app/components/encryptable/row.js

```diff
@@ -109,22 +135,26 @@ export default class RowComponent extends Component {
   }
 
   @action
-  showPassword() {
+  showAttribute(attribute) {
     this.fetchEncryptable();
-    this.isPasswordVisible = true;
+    this[
+      `is${attribute.charAt(0).toUpperCase()}${attribute.slice(1)}Visible`
+    ] = true;
 
-    this.passwordHideCountdownTime = new Date().getTime();
+    this.hideCountdownTime = new Date().getTime();
 
-    this.passwordHideTimerInterval = setInterval(() => {
+    this.hideTimerInterval = setInterval(() => {
       let now = new Date().getTime();
 
-      let passedTime = now - this.passwordHideCountdownTime;
+      let passedTime = now - this.hideCountdownTime;
 
       let passedTimeInSeconds = Math.floor(passedTime / 1000);
 
       if (passedTimeInSeconds >= this.HIDE_TIME) {
-        this.isPasswordVisible = false;
-        clearInterval(this.passwordHideTimerInterval);
+        this[
+          `is${attribute.charAt(0).toUpperCase()}${attribute.slice(1)}Visible`
```

Comment:
this hides the passwort after some time... seems overly complicated though... and should probably be extracted into a method if used several times